src/IIIFPresentation/Models/Database/Collections/Collection.cs

```diff
@@ -39,7 +39,7 @@ public class Collection
     public DateTime Modified { get; set; }
 
     /// <summary>
-    /// Who last committed a change to this Collection
+    /// Who last created this Collection
```

Comment:
maybe removing `last` would be good on this as well, as I don't think this can be changed